### PR TITLE
Remove runtime configuration

### PR DESCRIPTION
**Pull request by nvaytet**
Since all the configuration options for the plotting have been removed (plotting is now all done in Plopp), the configuration options are rather limited.
There is probably no one who is changing the colors of the buttons in jupyter lab.

To remove the `confuse` runtime dependency of scipp, we remove the configuration entirely.

### CODE REVIEW

File: src/scipp/visualizations/__init__.py
Comment (jl-wynen) [inline src/scipp/visualizations/__init__.py:1]: Why did you move code here as part of this PR?

File: src/scipp/visualizations/colors.py [hunk 1]
```diff
@@ -4,6 +4,17 @@
...
+STYLE = {
+    'attrs': '#ff5555',
+    'coords': '#c6e590',
+    'data': '#f6d028',
+    'masks': '#c8c8c8',
+    'button': '#bdbdbd49',  # Color for plot control button.
+    'button_selected': '#bdbdbdbb',  # Color for selected plot control button.
+    'header_text': '#111111',  # Color for table header text.
+    'hover': '#d6eaf8',  # Color for hovering on table rows.
+}
```
Comment (jl-wynen) [inline src/scipp/visualizations/colors.py:16]: Where is this used? Shouldn't this be part of Plopp?

**Review by jl-wynen — COMMENTED** [review]
What is you plan with plopp? Do you want to add a config file there?

**Comment by nvaytet** [comment]
> What is you plan with plopp? Do you want to add a config file there?

Not for now. No one has asked for one, so until then...

File: src/scipp/visualizations/colors.py (hunk 1 above)
Comment (nvaytet) [reply src/scipp/visualizations/colors.py:16]: I think this is used in the css style we are injecting in the html repr?

Comment (nvaytet) [reply src/scipp/visualizations/colors.py:16]: Here? https://github.com/scipp/scipp/pull/3254/files#diff-326b1107b97cc4ee3fa7eb1ccadb74e07065d5c0d7e532a0f0c67692602cc3e9R29

File: src/scipp/visualizations/__init__.py
Comment (nvaytet) [reply src/scipp/visualizations/__init__.py:1]: Because I thought it made more sense to group the visualizations together and put the stuff about colors there, since it is only used by the different visualizations?

File: src/scipp/visualizations/colors.py (hunk 1 above)
Comment (jl-wynen) [reply src/scipp/visualizations/colors.py:16]: Can you move those into the CSS. There is no reason to keep them in Python when we can't customise them anyway.

File: requirements/test.txt [hunk 2]
```diff
@@ -11,28 +11,28 @@ attrs==23.1.0
     # via hypothesis
 beautifulsoup4==4.12.2
     # via
-    #   -r test.in
+    #   -r requirements/test.in
```
Comment (SimonHeybrock) [inline requirements/test.txt:14]: I you run `pip-compile-multi` from within the `requirements` folder you won't get this update noise.

File: src/scipp/visualizations/__init__.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: BSD-3-Clause
```
Comment (SimonHeybrock) [inline src/scipp/visualizations/__init__.py:1]: Can you make this singular, `visualization`? I think having plural on module names is not so common?

File: src/scipp/visualizations/colors.py (hunk 1 above)
Comment (nvaytet) [reply src/scipp/visualizations/colors.py]: Can you point me to where that is? I can't seem to find the right place to put this. Thanks

Comment (jl-wynen) [reply src/scipp/visualizations/colors.py]: We load the style as a template and format it using this: https://github.com/scipp/scipp/blob/5b12dc349fe8f2086cd1e237f5bb4a04dd2f6a62/src/scipp/html/resources.py#L23
This is no longer needed, I think. So this file https://github.com/scipp/scipp/blob/main/src/scipp/html/templates/style.css.template can be turned into a regular CSS file and all placeholders (e.g. `$data_color`) can bereplaced with the colours you listed here.

Comment (nvaytet) [reply src/scipp/visualizations/colors.py]: But then if it is no longer a template, the `data_color` would have to be duplicated in the `colors.py` file (used by e.g. `sc.show`) and in the `style.css` file.

It seems that `button`, `button_selected`, and `hover` are not used in the `style.css.template` and that only `header_text` would be the one that lives only in `style.css`?

Comment (jl-wynen) [reply src/scipp/visualizations/colors.py]: > It seems that button, button_selected, and hover are not used in the style.css.template and that only header_text would be the one that lives only in style.css?

The buttons are from back when we had custom buttons on plots that looked a little different from the builtin ones. So this can probably be removed.

Do we actually still use colours in CSS? Or can we drop them with the newer simplified table views?

Comment (nvaytet) [reply src/scipp/visualizations/colors.py]: I guess the prefix `sc-table` in the css implies it's only used for the html table representation?

Comment (jl-wynen) [reply src/scipp/visualizations/colors.py]: Grepping for `sc-table` onyl gives me results in css. So the python code never even generates HTML elements with this class. But best double check the implementation before removing anything.

**Review by jl-wynen — APPROVED** [review]
